opentelemetry-exporter-google-cloud/src/opentelemetry/exporter/cloud_trace/__init__.py

```diff
@@ -361,36 +361,30 @@ def _extract_resources(resource: Resource) -> Dict[str, str]:
     }
 
 
-PREDIFINED_LABELS = [
-    "/agent",
-    "/component",
-    "/error/message",
-    "/error/name",
-    "/http/client_city",
-    "/http/client_country",
-    "/http/client_protocol",
-    "/http/client_region",
-    "/http/host",
-    "/http/method",
-    "/http/path",
-    "/http/redirected_url",
-    "/http/request/size",
-    "/http/response/size",
-    "/http/route",
-    "/http/status_code",
-    "/http/url",
-    "/http/user_agent",
-    "/pid",
-    "/stacktrace",
-    "/tid",
-]
-
-
-def _key_predefined(key: str):
-    result = "/%s" % key.replace(".", "/")
-    if result in PREDIFINED_LABELS:
-        return result
-    return key
+LABELS_MAPPING = {
+    # "agent": "/agent",
+    "component": "/component",
+    "error.message": "/error/message",
+    "error.name": "/error/name",
+    "http.client_city": "/http/client_city",
+    "http.client_country": "/http/client_country",
+    "http.client_protocol": "/http/client_protocol",
+    "http.client_region": "/http/client_region",
+    "http.host": "/http/host",
+    "http.method": "/http/method",
+    "http.path": "/http/path",
+    "http.redirected_url": "/http/redirected_url",
+    # https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/http.md#common-attributes
+    "http.request_content_length": "/http/request/size",
+    "http.response_content_length": "/http/response/size",
+    "http.route": "/http/route",
+    "http.status_code": "/http/status_code",
+    "http.url": "/http/url",
+    "http.user_agent": "/http/user_agent",
```

Comment:
A few tweaks and we should remove the ones not in the specification (https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/http.md)

```suggestion
    # this one might be http.flavor? I'm not sure
    "http.scheme": "/http/client_protocol",
    "http.host": "/http/host",
    "http.method": "/http/method",
    # https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/http.md#common-attributes
    "http.request_content_length": "/http/request/size",
    "http.response_content_length": "/http/response/size",
    "http.route": "/http/route",
    "http.status_code": "/http/status_code",
    "http.url": "/http/url",
    "http.user_agent": "/http/user_agent",
```